src/main/kotlin/tech/pegasys/libp2p/noiseintegration/NoiseOverTcp.kt

```diff
@@ -0,0 +1,34 @@
+package tech.pegasys.libp2p.noiseintegration
```

Comment:
M.b. move this file to `tests` ?

Reply:
I've removed this file and `NoiseOverTcpTest.kt` since the test that uses Noise with embedded channels is in `NoiseSecureChannelTest`, which is in `test`.